### PR TITLE
preferences: only text should be translatable

### LINKED ISSUE #7467: it terminated and the interface setting appears to be lost
Describe the bug
After a few minutes of work, when using the "tone mapping" tool, the application closed.
This was the screen when I restarted the application.
The interface setting appears to be lost and the application also no longer responds correctly to commands.
From further tests made, the behavior is repeated at each modification of the application settings.
The application also no longer responds correctly to commands.

To Reproduce
I don't understand how to reproduce the behavior as it occurs every time the application settings are changed.

Expected behavior
The interface setting appears to be lost and the application also no longer responds correctly to commands.
From further tests made, the behavior is repeated at each modification of the application settings.
The application also no longer responds correctly to commands.

Screenshots
Attached

Platform:
 - Darktable Version: 3.4
 - OS: Windows 10 Pro 20H2 Build 19042.685
AMD Ryzen 7 1800X
32GB RAM
Radeon RX560 4GB Driver 20.11.3
Windows 10 Pro 20H2 Build 19042.685
 OpenCL activated or no? --> NO, the same behavior also occurs after activation

Additional context:
Going to the path "C:\Users\xxx\AppData\Local\darktable" and deleting all the configuration files in the folder, DarkTable returns to work correctly, a reset is performed.

### PR DESCRIPTION
**Pull request by elstoc**
resolves #7467

### CODE REVIEW

**Comment by elstoc** [comment]
@TurboGit I think this is probably a good candidate for a 3.6.1 release - it's an issue I've seen reported a few times.

**Comment by parafin** [comment]
Translation must not include */ after this change - won't "fuzzy" translations of old strings make it quite probable?

**Comment by elstoc** [comment]
obviously the alternative is to fix the translations themselves but we can't guarantee this error won't creep back in again

File: src/gui/preferences.c [hunk 1]
```diff
@@ -491,14 +491,14 @@ static void init_tab_general(GtkWidget *dialog, GtkWidget *stack, dt_gui_themetw
     else
     {
       //load default text with some pointers
-      gtk_text_buffer_set_text(buffer, _("/* ERROR Loading user.css */"), -1);
+      gtk_text_buffer_set_text(buffer, g_strconcat("/* ", _("ERROR Loading user.css"), " */", NULL), -1);
```
Comment (johnny-bit) [inline src/gui/preferences.c:494]:
```suggestion
      gchar* css_error_msg = g_strconcat("/* ", _("ERROR Loading user.css"), " */", NULL);
      gtk_text_buffer_set_text(buffer, css_error_msg, -1);
      g_free(css_error_msg);
```

File: src/gui/preferences.c [hunk 2]
```diff
@@ -491,14 +491,14 @@ static void init_tab_general(GtkWidget *dialog, GtkWidget *stack, dt_gui_themetw
...
     }
     g_free(usercsscontent);
   }
   else
   {
     //load default text
-    gtk_text_buffer_set_text(buffer, _("/* Enter CSS theme tweaks here */\n\n"), -1);
+    gtk_text_buffer_set_text(buffer, g_strconcat("/* ", _("Enter CSS theme tweaks here"), " */\n\n", NULL), -1);
```
Comment (johnny-bit) [inline src/gui/preferences.c:501]:
```suggestion
    gchar* css_prompt = g_strconcat("/* ", _("Enter CSS theme tweaks here"), " */\n\n", NULL);
    gtk_text_buffer_set_text(buffer, css_prompt, -1);
    g_free(css_prompt);
```

**Review by johnny-bit — CHANGES_REQUESTED** [review]
`g_strconcat` results need freeing ;)

also - it'd be great to fix the translation files that have this error, just so that "fuzzy" translations won't catch that.

**Comment by elstoc** [comment]
It's possible that the broken translation itself has been fixed because I can't find an error in the po files. Still (I think) better to translate just the words if we can.

**Review by TurboGit — APPROVED** [review]
All good, thanks !

**Pull request merged**